src/config.rs

```diff
@@ -117,6 +120,21 @@ pub struct Config {
     pub connection_window_size: u32,
     pub frame_size: u32,
 
+    // The limit of how many streams a single HBONE pool connection will be limited to, before
+    // spawning a new conn rather than reusing an existing one, even to a dest that already has an open connection.
+    //
+    // This can be used to effect flow control for "connection storms" when workload clients
+    // (such as loadgen clients) open many connections all at once.
+    //
+    // Note that this will only be checked and inner conns rebalanced accordingly when a new connection
```

Comment:
What does "rebalanced" mean? that seems impossible to me.

Reply:
Old wording, will remove.